src/main/java/org/elasql/perf/tpart/workload/TransactionFeatures.java

```diff
@@ -16,8 +18,37 @@
 	// Defines a read-only list for feature keys
 	public static final List<String> FEATURE_KEYS;
 	
+	private static class FeatureKeys {
+		private List<String> featureKeys = new ArrayList<String>();
+		private int serverCount; 
+		
+		public FeatureKeys(int serverCount) {
+			this.serverCount = serverCount;
+		}
+		
+		public boolean add(String key) {
+			return featureKeys.add(key);
+		}
+		
+		public void addWithServerCount(String key) {
+			for (int serverId = 0; serverId < serverCount; serverId++) {
+				String keyWithServerId = getKeyWithServerId(key, serverId);
+				featureKeys.add(keyWithServerId);
+			}
+		}
+		
+		public List<String> getList() {
+			return featureKeys;
+		}
+	}
```

Comment:
It is a good attempt to construct the list of feature keys from a class, but I don't see there is a need to do this. I think this can be replaced by a simple function: `addKeysWithServerCount(List<String> list, String keyName)`, and the code in the static block may become like this:

```java
static {
    List<String> featureKeys = new ArrayList<String>();
    
    ....

    addKeysWithServerCount(featureKeys, "System CPU Load");
    ....
}
```

Reply:
OK.